frontend/src/components/HelpButton.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-dialog v-model="dialog" width="60%" :theme="layoutTheme">
+    <template v-slot:activator="{ props }">
+      <v-btn
+        :height="44"
+        :width="44"
+        v-bind="props"
+        icon="mdi-help"
+        color="primary"
+        class="help"
+      />
+    </template>
+
+    <v-card>
+      <v-toolbar title="Instructions">
+        <template #append>
+          <v-btn @click="dialog = false" icon="mdi-close"></v-btn>
+        </template>
+      </v-toolbar>
+      <v-card-text>
+        <h2 class="text-h6 mb-2 mt-3">This Playground supports the following kinds of rules</h2>
+
+        <ul class="pl-4 my-2">
+          <li>Validation Rules</li>
+          <li>Mutation Rules</li>
+          <li>Geneneration Rules</li>
+          <li>ImageVerification Rules</li>
+        </ul>
+
+        <p>
+          Not possible is to access private registries for e.g. ImageVerification rules, also not supported is the "clone" functionality of generate rules.
+        </p>
+
+        <h2 class="text-h6 mb-2 mt-3">Context and Variables</h2>
+
+        <p>
+          Currently it is not possible to add variables from resources
+          or do actual APICalls, instead its possible to mock variables over the
+          "variables" configuration in the "context" input
+        </p>
+
+        <h2 class="text-h6 mb-2 mt-3">Multiple Manifests</h2>
+
+        <p>
+          It is supported to define multiple policies and/or resources as input,
+          the context as well as variables will be shared and used for all executations.
```

Comment:
"It is supported to define multiple policies and/or resources as inputs. Context and variables will be shared for all executions."